pkg/operator/k8sutil/cmdreporter/cmdreporter.go

```diff
@@ -92,22 +92,10 @@ func New(
 	cmd, args []string,
 	rookImage, runImage string,
 ) (*CmdReporter, error) {
-	if clientset == nil || ownerRef == nil {
```

Comment:
why not keep the error checking first? maybe i'm missing what changed when it moved down

Reply:
Blaine is validating the cfg struct to catch potential errors in its construction (see comment on (new) #107).